src/content/blog/2024_mitiq_impact.md

```diff
@@ -0,0 +1,147 @@
+---
+title: The impact of Mitiq
+author: nate stemen
+day: 24
+month: 12
+year: 2024
+tags: 
+  - python
+  - mitiq
+  - community
+---
+
+As we approach the 5 year anniversary since the Mitiq project has begun, we're starting to see how the project has played a role in shaping the ecosystem.
```

Comment:
```suggestion
As we approach the 5-year anniversary since the Mitiq project has begun, we're starting to see how the project has played a role in shaping the ecosystem.
```
This sentence makes it sound like we see the impact only now after 5 years, which sounds too negative imo.

Reply:
Good point. I'll rework this to say something more like "the five-year anniversary provides an opportunity to reflect how this project has shaped the ecosystem".